### PR TITLE
shiva,nova content fill

### PR DESCRIPTION
**Pull request by OttoChen312**
## Description

[filled the content for the shiva and nova projects according to google docs (still a problem of the letter L appearing Infront of image descriptions)]

## Updates

[made changes for ProjectWrapper, ProjectDefault, and ProjectJoin specifically added if statements for when the project.name is shiva or nova in ProjectDefault and ProjectJoin. Added images for shiva and nova projects]

## Checklist

- [ ] Ran `npm run lint:fix` to format code
- [ ] Requested at least one reviewer
- [ ] Connected PR to Trello ticket (steps below)
- [ ] Addressed code review comments
- [ ] Gained at least one approval for your PR

## Connecting your PR to the corresponding ticket:
- Click on the "GitHub" button found on the right of your Trello ticket
- Choose "Attach PR" from the dropdown list
- Link your GitHub account (if you haven't done so already)
- Navigate to`VCL-content-platform` and choose your PR from the dropdown.

### CODE REVIEW

**Comment by OttoChen312** [comment]
[shiva,nova content fill](https://trello.com/c/hPyLzj9U/168-shivanova-content-fill)

File: package.json
Comment (ElisaHu1) [inline package.json:1]: please revert this change to export SET

File: .idea/.gitignore
Comment (ElisaHu1) [inline .idea/.gitignore:1]: do not push .idea change to github

File: .idea/vcs.xml
Comment (ryanyae) [inline .idea/vcs.xml:1]: All these .idea/... files seem to be specific for your IDE, and you should refrain from pushing this into main.

File: package-lock.json
Comment (ryanyae) [inline package-lock.json:1]: You should also refrain from pushing package-lock.json files, unless you have downloaded a new library for your ticket.

File: src/pages/Project/ProjectJoin.tsx
Comment (ryanyae) [inline src/pages/Project/ProjectJoin.tsx:1]: This is not the place for content to be filled. I know that the code base is very confusing, but all this information should actually be filled in within the ProjectWrapper.tsx file. This is because we want to keep we actually want to keep all the components of Project as dynamic as possible.

File: src/pages/Project/ProjectJoin.tsx [hunk 1]
```diff
@@ -134,6 +161,42 @@ const ProjectJoin: React.FC<ProjectProps> = (props) => {
...
+                        </div>
+                        <div className='project-join-card'>
+                            <h5>{'Does Consciousness Matter? Noticeable Edits on Overlearned Cues and the Effect on Blindness Rates and Cueing Effects Between Seen and Unseen Cues'}</h5>
+                            <p>{'Author: Isha Verma'}</p>
+                            <p>{'Sponsoring Research Assistant: Ying Zeng'}</p>
+                            <p>{'Supervisor: Dr. Ronald Rensink'}</p>
+                        </div>
+                    </div>
+                </section>
+            )}
```
Comment (ryanyae) [inline src/pages/Project/ProjectJoin.tsx:199]: This whole NOVA and SHIVA section is very static and like I said before, we want to keep it dynamic because all the projects have the same information. So if we were to keep all the descriptions in ProjectWrapper.tsx we can just manipulate the props.

File: src/pages/Project/ProjectJoin.tsx [hunk 2]
```diff
@@ -143,7 +206,15 @@ const ProjectJoin: React.FC<ProjectProps> = (props) => {
...
         </section> */}
+            {props.project.name == 'SHIVA' && (
+                <ProjectGallery displayNumber={4} compTitle={'Testimony'} itemArray={shivaList} />
+            )}
+            {props.project.name == 'NOVA' && (
+                <ProjectGallery displayNumber={4} compTitle={'Testimony'} itemArray={novaList} />
+            )}
+            {props.project.name !== 'SHIVA' && props.project.name !=='NOVA'&&(
             <ProjectGallery displayNumber={4} compTitle={'Testimony'} itemArray={dummyList} />
+                )}
```
Comment (ryanyae) [inline src/pages/Project/ProjectJoin.tsx:217]: Applies to here as well, what I mentioned before about being the page being dynamic.

File: src/statics/images/nova/nova3.png
Comment (ryanyae) [inline src/statics/images/nova/nova3.png:1]: I was just wondering if we could crop out the red YouTube bar at the bottom of the page.

File: src/pages/Project/ProjectDefault.tsx
Comment (ryanyae) [inline src/pages/Project/ProjectDefault.tsx:1]: Applies to here as well, what I mentioned below about keeping everything dynamic.

**Review by ryanyae — COMMENTED** [review]
Good work Otto, I do understand that the code base can be very confusing but I strongly encourage to keep in mind about utilizing React's dynamic nature. Whenever you try and write code just try and think about how you could make optimal use of props if you are to reuse certain React components. Don't worry about this ticket, I strongly recommend you to  try and draw out how the code looks like through a UML diagram! Thank you for your hard work none the less!
